elf/elf.py

```diff
@@ -153,6 +153,16 @@ def add_symbols_version(self):
         # symbols_version_requirement
         pass
 
+	def add_functions(self):
```

Comment:
I believe this section of code uses tabs instead of spaces. Would it be possible to use four spaces instead per tab instead?

Reply:
Of course no problem and it's done